example/session/src/main/java/org/apache/iotdb/SessionExample.java

```diff
@@ -828,18 +828,19 @@ private static void nonQuery() throws IoTDBConnectionException, StatementExecuti
     session.executeNonQueryStatement("insert into root.sg1.d1(timestamp,s1) values(200, 1)");
   }
 
-  private static void setTimeout() throws StatementExecutionException {
-    Session tempSession = new Session(LOCAL_HOST, 6667, "root", "root", 10000, 20000);
-    tempSession.setQueryTimeout(60000);
+  private static void setTimeout() throws StatementExecutionException, IoTDBConnectionException {
+    try (Session tempSession = new Session(LOCAL_HOST, 6667, "root", "root", 10000, 20000)) {
+      tempSession.setQueryTimeout(60000);
+    }
   }
 
   private static void createClusterSession() throws IoTDBConnectionException {
```

Comment:
We can just remove this function now as it's used for old cluster module

Reply:
ok

Reply:
do not remove `setTimeout`, remove `createClusterSession`

Reply:
ok